modules/claims_api/app/workers/claims_api/report_unsuccessful_submissions.rb

```diff
@@ -72,14 +73,31 @@ def claims_totals
     end
 
     def poa_totals
-      totals = ClaimsApi::PowerOfAttorney.where(created_at: @from..@to).group(:status).count
-      total_submissions = totals.sum { |_k, v| v }
-      totals.merge(total: total_submissions).deep_symbolize_keys
+      @poa_consumers.map do |cid|
+        counts = ClaimsApi::PowerOfAttorney.where(created_at: @from..@to, cid: cid).group(:status).count
+        totals = counts.sum { |_k, v| v }
+
+        if totals.positive?
+          consumer_name = ClaimsApi::CidMapper.new(cid: cid).name
+          {
+            consumer_name => counts.merge(totals: totals)
+                                   .deep_symbolize_keys
+          }
+        end
+      end
     end
 
     def unsuccessful_poa_submissions
```

Comment:
Since the output of this method has changed, I think we'll have to update the "[preview mock](https://github.com/department-of-veterans-affairs/vets-api/blob/API-17762-POA-email-report/spec/mailers/previews/claims_api_unsuccessful_report_mailer_preview.rb#L52-#L59)" and the actual "[_poa_errors_table.html.erb](https://github.com/department-of-veterans-affairs/vets-api/blob/API-17762-POA-email-report/modules/claims_api/app/views/claims_api/unsuccessful_report_mailer/_poa_errors_table.html.erb)" view.